src/components/common/Posts/Post.jsx

```diff
@@ -15,6 +15,7 @@ import {doc, getDoc, updateDoc} from 'firebase/firestore';
 import { useUserData } from 'context/UserContext';
 import { GetIndividualUserData } from 'utils/authService';
 import { Link, NavLink } from 'react-router-dom';
+import { Toast } from 'components/UI/Toast/Toast';
```

Comment:
Same here. Create an index file in components and export from there. Here, it will be import {Toast} from 'components'

Reply:
done